requirements.yml

```diff
@@ -16,11 +16,19 @@
 - src: andrewrothstein.miniconda
   version: "v5.2.6"
 
-- src: nvidia.nvidia_driver
-  version: "v1.1.0"
+  ## - src: nvidia.nvidia_driver
+  ## version: "v1.1.0"
+  ## 
+  ## - src: nvidia.nvidia_docker
+  ## version: "v1.1.0"
 
-- src: nvidia.nvidia_docker
-  version: "v1.1.0"
+- src: https://github.com/phogan-nvidia/ansible-role-nvidia-driver
+  name: "nvidia.nvidia_driver"
+  version: "proxy"
+
+- src: https://github.com/phogan-nvidia/ansible-role-nvidia-docker
+  name: "nvidia.nvidia_docker"
+  version: "proxy"
```

Comment:
Before merging this into DeepOps we should merge in the PRs for these roles. I'll review these as well.

Reply:
Ok, changes in the role repositories are merged in! Instead of pointing to your github, we can point to the v1.2.0 releases for each of these roles.

Reply:
Great.